src/pages/overview/pdf-services-api/howtos/ocr-pdf.md

```diff
@@ -29,40 +29,44 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
 // Run the sample:
 // mvn -f pom.xml exec:java -Dexec.mainClass=com.adobe.pdfservices.operation.samples.ocrpdf.OcrPDF
  
- public class OcrPDF {
+public class OcrPDF {
  
   // Initialize the logger.
   private static final Logger LOGGER = LoggerFactory.getLogger(OcrPDF.class);
  
   public static void main(String[] args) {
- 
-   try {
- 
-    // Initial setup, create credentials instance.
-    Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-        .withClientId("PDF_SERVICES_CLIENT_ID")
-        .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-        .build();
- 
-    //Create an ExecutionContext using credentials and create a new operation instance.
-    ExecutionContext executionContext = ExecutionContext.create(credentials);
-    OCROperation ocrOperation = OCROperation.createNew();
- 
-    // Set operation input from a source file.
-    FileRef source = FileRef.createFromLocalFile("src/main/resources/ocrInput.pdf");
-    ocrOperation.setInput(source);
- 
-    // Execute the operation
-    FileRef result = ocrOperation.execute(executionContext);
- 
-    // Save the result at the specified location
-    result.saveAs("output/ocrOutput.pdf");
- 
-   } catch (ServiceApiException | IOException | SdkException | ServiceUsageException ex) {
-    LOGGER.error("Exception encountered while executing operation", ex);
-   }
+        try {
+             // Initial setup, create credentials instance
+            Credentials credentials = new ServicePrincipalCredentials(
+                    System.getenv("PDF_SERVICES_CLIENT_ID"),
+                    System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+            // Creates a PDF Services instance
+            PDFServices pdfServices = new PDFServices(credentials);
+
+            // Creates an asset from source file and upload
+            InputStream inputStream = Files.newInputStream(new File("src/main/resources/ocrInput.pdf").toPath());
+            Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
+
+            // Creates a new job instance
+            OCRJob ocrJob = new OCRJob(asset);
+
+            // Submits the job and gets the job result
+            String location = pdfServices.submit(ocrJob);
+            PDFServicesResponse<OCRResult> pdfServicesResponse = pdfServices.getJobResult(location, OCRResult.class);
+
+            // Get content from the resulting asset(s)
```

Comment:
same

Reply:
As discussed, IG this is from already reviewed samples.